src/wp-includes/script-loader.php

```diff
@@ -1843,6 +1843,32 @@ function wp_just_in_time_script_localization() {
 	);
 }
 
+
+/**
+ * Prints a loader script if there is text/template registered script.
+ *
+ * When added to the DOM, this script converts any text/template scripts
+ * associated with a handle into type/javascript, and executes them.
+ */
+function wp_print_template_loader_script() {
```

Comment:
IMO, this could be more clearly named `wp_print_delayed_inline_script_loader`. Looking at the name otherwise makes me think it is about print a script that loads in some Mustache template or something like that.

Reply:
@westonruter this was renamed in a recent commit as per your suggestion (thank you) and references updated.